### PR TITLE
[hotfix][docs] Remove unnecessary html link tags and fix typo

### PR DESCRIPTION
**Pull request by leonardBang**
* 1 Remove unnecessary html link tags
* 2 Fix typo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
-# FLink Table Store
+# Flink Table Store
 
 Flink Table Store is a unified streaming and batch store for building dynamic tables on Apache Flink.
 
 Flink Table Store is developed under the umbrella of [Apache Flink](https://flink.apache.org/).
 
-## <a name="build"></a>Building the Project
```
Comment (JingsongLi) [inline README.md:7]: It is better to leave this, the title can be `build`.

File: README.md [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
 
-## <a name="contributing"></a>Contributing
+## Contributing
 
 You can learn more about how to contribute on the [Apache Flink website](https://flink.apache.org/contributing/how-to-contribute.html). For code contributions, please read carefully the [Contributing Code](https://flink.apache.org/contributing/contribute-code.html) section for an overview of ongoing community work.
 
-## <a name="license"></a>License
+## License
 
-The code in this repository is licensed under the [Apache Software License 2](LICENSE).
```
Comment (JingsongLi) [inline README.md:19]: `Apache Software License` is ASL. https://wiki.debian.org/DFSGLicenses#The_Apache_Software_License_.28ASL.29

**Review by JingsongLi — COMMENTED** [review]
Thanks @leonardBang for your contribution, left comments.

File: README.md (hunk 1 above)
Comment (leonardBang) [reply README.md:7]: In fact you do not need this, the `Building the Project` is not to long, or you can use `Build` as the title for unification with other tiles.

File: README.md (hunk 2 above)
Comment (leonardBang) [reply README.md:19]: Just let us keep the readme same with our license.
image

File: README.md (hunk 1 above)
Comment (leonardBang) [reply README.md:7]: I make it shorter, sir

File: README.md (hunk 2 above)
Comment (leonardBang) [reply README.md:19]: well, I think ASL is fine to me, too. I revert the change.

File: README.md (hunk 1 above)
Comment (JingsongLi) [reply README.md:7]: shorter is good (:

**Review by JingsongLi — APPROVED** [review]
+1

**Pull request merged**